citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.html

```diff
@@ -5,6 +5,15 @@
 .popover {
 	min-width: 300px;
 }
+
+.rotating {
+    -webkit-transform: rotate(360deg);
+    -webkit-transition-duration: 1s;
+    -webkit-transition-delay: now;
+    -webkit-animation-timing-function: linear;
+    -webkit-animation-iteration-count: infinite;
+}
```

Comment:
let's have adhvik put this class into the icon set so it can be used everywhere.